added runtime requirements and caveats (#60)
mulesoft/docs-visualizer: modules/ROOT/pages/cloudhub-setup.adoc

== Verify Supported Mule Versions for Policies

> 

Reviewer: I think the "or later" covers the +. Remove the plus?

> The APIs and policies that Anypoint Visualizer displays in the Policy visualization come from an update to the agent in Mule runtime engine (Mule). As a result, policies are shown in Anypoint Visualizer only for applications running on any of the patches of Anypoint Visualizer supported Mule versions available on December 12, 2019 or later.

Reviewer: Should visualization be init cap?
I had to read the 2nd sentence twice. Maybe "on Mule version patches supported by Anypoint Visualizer that were released on xxx and later"?

Reviewer: No, that doesn't work either, I see.
I would say add a hyphen after Visualizer (Visualizer-supported), but I think we can't do that.
"Anypoint Visualizer shows policies only for apps running on supported Mule versions released on xxxx and later."

Do customers know how to get the subset of Mule versions that support Visualizer?

Author: ```suggestion
The APIs and policies that Anypoint Visualizer displays in the Policy visualization come from an update to the agent in Mule runtime engine (Mule). As a result, Anypoint Visualizer shows policies only for apps running on supported Mule versions released on December 12, 2019 or later.
```
